pyproject.toml

```diff
@@ -0,0 +1,4 @@
+[tool.astropy-bot]
+
+[tool.astropy-bot.changelog_checker]
+filename = "CHANGES.rst"
```

Comment:
I think this is the default?

Reply:
better be safe than sorry :) 

(I took astroquery's configs and updated them accordingly rather than trying to think hard about it 😅 )